agent/bench-scripts/pbench-dbench

```diff
@@ -318,6 +318,8 @@ max_key_length=0
 export benchmark config
 pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir --sysinfo=$sysinfo beg
 pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir beg
+# on abnormal exit, make sure that the metadata log exists and is complete.
+trap "pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir int; mv $benchmark_iterations $pbench_run" INT QUIT
```

Comment:
Can we consider changing the definition of `${benchmark_iterations}` so that it is kept only in `${pbench_run}`?

Reply:
@portante: I think it should be `${benchmark_run_dir}`, not `${pbench_run}` - correct? Something like this:


    benchmark_iterations="${benchmark_run_dir}/${benchmark_fullname}.iterations"

Do we want the file name to contain the full benchmark name, or to be simply `iterations`?

Reply:
Seems like having it named `${benchmark_run_dir}/.iterations` would keep it from conflicting but make sure it is not prominent.